src/DIRAC/Resources/Computing/GlobusComputingElement.py

```diff
@@ -27,10 +27,22 @@
 from DIRAC.FrameworkSystem.Client.ProxyManagerClient import gProxyManager
```

Comment:
Do not spend time on this CE. Honestly, it could even be removed.

Reply:
Ok I deleted it